frontend/src/components/TextSizeToggle.js

```diff
@@ -0,0 +1,5 @@
+const toggleFontSize = () => {
```

Comment:
we are probably going to need to take this out, you can maybe cherry pick the commits and get rid of the commits that add the redundancy

Reply:
Yep taking it out sounds good. Pretty sure the commit that added this only added this along eith the popup window test file so it should be fine